src/components/radio/Radio.tsx

```diff
@@ -1,11 +1,11 @@
-import { Component, createSignal, Show } from 'solid-js'
+import { Component, createEffect, createSignal } from 'solid-js'
 
 import StyledIcon from './StyledIcon'
 import StyledInput from './StyledInput'
 import StyledLabel from './StyledRadio'
 
-export type IRadioProps = {
-  label?: any
+export type RadioProps = {
+  label?: JSX.Element | string
```

Comment:
Btw, JSX.Element is:
```ts
type Element =
    | Node
    | ArrayElement
    | FunctionElement
    | string
    | number
    | boolean
    | null
    | undefined;
```
There's no need to do `| string`
But that's fine

Reply:
Actually, it was throwing error that string is not assignable to Element then I import `import type { JSX } from 'solid-js'` and it worked.